odl/test/trafos/fourier_test.py

```diff
@@ -46,6 +46,20 @@ def _params_from_dtype(dtype):
     return halfcomplex, complex_dtype(dtype)
 
 
+def _dft_space(shape, dtype='float64'):
```

Comment:
This seems to replaces ``odl.discr_sequence_space``. Was the other one used only here? Is there no reason to keep such a space/helper somewhere?

Edit: in ``odl/trafos/fourier``, the sequence space seems to have been replaced by uniform discretization. Can this be done here too?

Reply:
I've basically degraded `discr_sequence_space` to a test helper and inlined the code in `fourier.py` where it is used twice. The main reason for doing that is that for its name "sequence space" it was just a hack using a discrete function space which happened to yield the same results.
If we ever feel the need to make such a helper available we could revive it somewhere in `odl.util`, but for an "official" discrete space API, I think it was too hacky and not useful enough.